rclpy/executors/test/test_copyright.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_copyright.main import main
+
+
+def test_copyright():
+    root_path = os.path.realpath(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
As mentioned in https://github.com/ros2/examples/pull/182#discussion_r137910589 there is no reason for this complex code.

Same below.

Reply:
440015046c8e775b56b80006dc410f24d064be24